src/main/scala/com/codacy/client/stash/Repository.scala

```diff
@@ -4,7 +4,7 @@ import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
 case class Repository(slug: String, name: String, scm: String, public: Boolean,
-                      cloneUrl: String, cloneUrls: Seq[RepositoryUrl])
+                      cloneUrls: Seq[RepositoryUrl])
```

Comment:
not mandatory? but it's 0 or 1 right? wouldn't a option make more sense then? Or can be more than 1?